islandora-connector-broadcast/build.gradle

```diff
@@ -0,0 +1,29 @@
+apply plugin: 'osgi'
+
+description = 'Islandora CLAW Broadcaster'
+
+dependencies {
+    compile group: 'org.apache.camel', name: 'camel-core', version: camelVersion
+    compile group: 'org.apache.camel', name: 'camel-blueprint', version: camelVersion
+    compile group: 'org.slf4j', name: 'slf4j-api', version: slf4jVersion
```

Comment:
It's fine to leave this here, but I'd note that you're not actually using SLF4J in the code. (You may want to at some future point, but I'd probably remove this until that time comes).